website/static/favicon/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what is the use of this file @bmlancien ?

Reply:
I think this is for displaying a favicon for IE on WIndows 8 and 10